licenserc.toml

```diff
@@ -0,0 +1,55 @@
+headerPath = "scripts/license-header.txt"
+
+excludes = [
+    # Bazel files
+    ".bazelignore",
+    ".bazelrc",
+    ".bazelversion",
+    "BUILD.bazel",
+    "MODULE.bazel.lock",
+    "WORKSPACE",
+
+    # Root license file is more verbose
+    "LICENSE",
+
+    # Ignore scripts that drive releases
+    "scripts/*",
+
+    # Configuration files
+    "*.conf",
+    "*.toml",
+    "*.yaml",
+    "*.yml",
+
+    # Documentation files
+    "*.md",
+
+    # Pkl generated files
+    'PklProject.deps.json',
+
+    # Test specification files
+    "tests/**/*.txt",
+]
+
+[git]
+attrs = 'auto'
+ignore = 'auto'
+
+[properties]
+inceptionYear = 2024
+copyrightOwner = "Apple Inc. and the Pkl project authors"
+
+[mapping.DOUBLESLASH_STYLE]
```

Comment:
It'd be good to keep the current header style for Pkl, especially because doc comments would clash.

I played around with this and created a header style that seems to work:

```toml
[PKL_STYLE]

firstLine = "//===----------------------------------------------------------------------===//"

endLine = "//===----------------------------------------------------------------------===//"

beforeEachLine = "// "

firstLineDetectionPattern = "//==="

lastLineDetectionPattern = "//==="
```

This needs to be a separate file, and you'd set `additionalHeaders` in the `licenserc.toml`.